securedrop/journalist.py

```diff
@@ -321,7 +321,7 @@ def index():
         else:
             unstarred.append(source)
         source.num_unread = len(
-            Submission.query.filter(Submission.source_id == source.id, Submission.downloaded == False).all())
+            Submission.query.filter(Submission.source_id == source.id, Submission.downloaded is False).all())
```

Comment:
If one wants to specifically check for False, one wants "x is False". There exists only 1 False object.

If one wants to check for any falsish value (False, 0, [], ...), then "not x" is the right way.

x == False (or x == True) is considered bad style usually and even some pep8 checkers complain about it.